core/src/main/resources/swagger/data-repo-only.yaml

```diff
@@ -115,29 +69,24 @@ paths:
             '*/*':
               schema:
                 $ref: '#/components/schemas/ErrorReport'
+      security:
```

Comment:
This authorization block is the same for the `post` and `get` endpoints here, does the openapi spec let you pull this out a level?

Reply:
I don't believe we can life it up one level, but we *should* put it at the top level as a default for all paths, then override the handful of paths that are different. I'd like to save that for a larger refactor, once the main api-docs.yaml is fixed up.